topomodelx/nn/cell/can_layer.py

```diff
@@ -310,25 +316,29 @@ def reset_parameters(self) -> None:
     def forward(self, x_0, lower_neighborhood, upper_neighborhood) -> tuple[torch.Tensor, torch.Tensor, torch.Tensor]:  # type: ignore[override]
         r"""Forward pass.
 
-        .. math::
-            \begin{align*}
-            &🟥 \quad m_{x}^{(r)}
-                = \gamma^t(h_x^t) = \tau^t (a^t\cdot h_x^t)\\
-            &🟦 \quad h_x^{t+1,(r)}
-                = \phi^t(h_x^t, m_{x}^{(r)}), \forall x\in \mathcal C_r^{t+1}
-            \end{align*}
-
         Parameters
         ----------
-        x_0: torch.Tensor
-            Node signal of shape (num_nodes, in_channels_0).
-        neighborhood_0_to_0: torch.Tensor
-            Neighborhood matrix of shape (num_edges, 2).
+        x_0: torch.Tensor, shape = (num_nodes, in_channels_0)
```

Comment:
Most of the time you use square brackets for the shape, here normal brackets. We should decide on one and be consistent with it.

Reply:
Just tagging @jarpri so that it noted.